VideoLocker/res/layout/row_discussion_thread.xml

```diff
@@ -23,67 +27,63 @@
                 android:id="@+id/discussion_post_pin_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/x_small_icon_margin"
-                android:layout_marginRight="@dimen/x_small_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
                 app:iconName="fa-thumb-tack" />
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/discussion_post_following_icon"
-                android:layout_width="@dimen/edx_x_small"
-                android:layout_height="@dimen/edx_x_small"
-                android:layout_marginEnd="@dimen/x_small_icon_margin"
-                android:layout_marginRight="@dimen/x_small_icon_margin"
-                app:iconColor="@color/edx_grayscale_neutral_base"
+                style="@style/discussion_thread_icon"
+                app:iconColor="@color/edx_grayscale_neutral_dark"
                 app:iconName="fa-star" />
 
             <TextView
-                android:id="@+id/discussion_post_author"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:textColor="@color/edx_grayscale_neutral_base"
-                android:textSize="@dimen/edx_x_small"
-                tools:text="By Staff" />
+                android:id="@+id/discussion_post_replies_count"
+                style="@style/discussion_responses_small_text"
+                android:layout_marginEnd="@dimen/discussion_posts_icon_margin"
+                android:layout_marginRight="@dimen/discussion_posts_icon_margin"
+                tools:text="| Total: 20" />
+
+            <TextView
+                android:id="@+id/discussion_post_date"
+                style="@style/discussion_responses_small_text"
+                tools:text="| Last post: Apr 06, 2016" />
 
         </LinearLayout>
 
     </LinearLayout>
 
     <TextView
-        android:id="@+id/discussion_post_indicator_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:textSize="@dimen/edx_small"
-        android:layout_margin="@dimen/edx_margin"
-        android:drawablePadding="3sp"
-        tools:text="42" />
+        android:id="@+id/discussion_unread_replies_text"
+        style="@style/discussion_title_text"
+        fontPath="fonts/OpenSans-Semibold.ttf"
+        android:layout_marginEnd="@dimen/edx_margin"
+        android:layout_marginLeft="@dimen/edx_margin"
+        android:layout_marginRight="@dimen/edx_margin"
+        android:layout_marginStart="@dimen/edx_margin"
+        android:layout_marginTop="@dimen/discussion_posts_icon_margin"
+        android:textColor="@color/edx_brand_primary_base"
+        android:textStyle="bold"
```

Comment:
It would decouple the style from the layout, and make it easier to find and edit. I suggested it because it's a distinctive element with a custom font, but I suppose the final decision is up to you.